integreat_cms/release_notes/current/unreleased/3078.yml

```
@@ -0,0 +1,2 @@
+en: Sensible error message when adding inactive users to pages
+de: Aussagekräftige Fehlermeldung, wenn inaktive Nutzer:innen zu Seiten hinzugefügt werden
```

```suggestion
de: Zeige aussagekräftige Fehlermeldung, wenn inaktive Nutzer:innen zu Seiten hinzugefügt werden
```
Nit: We usually try to use the imperative form in our release notes :)